src/Avalonia.Base/PriorityValue.cs

```diff
@@ -52,6 +52,8 @@ internal class PriorityValue
             _validate = validate;
         }
 
+        public bool IsAnimating => ValuePriority < 0 && GetLevel(ValuePriority).ActiveBindingIndex != -1;
```

Comment:
We probably need more docs on priority value system, since this line has some "magic" numbers and it's hard to understand what it actually checks